thoth-api/src/model/mod.rs

```diff
@@ -215,3 +362,126 @@ macro_rules! db_insert {
         }
     };
 }
+
+#[test]
+fn test_doi_default() {
+    let doi: Doi = Default::default();
+    assert_eq!(doi, Doi { 0: "".to_string() });
+}
+
+#[test]
+fn test_orcid_default() {
+    let orcid: Orcid = Default::default();
+    assert_eq!(orcid, Orcid { 0: "".to_string() });
+}
+
+#[test]
+fn test_timestamp_default() {
+    let stamp: Timestamp = Default::default();
+    assert_eq!(
+        stamp,
+        Timestamp {
+            0: TimeZone::timestamp(&Utc, 0, 0)
+        }
+    );
+}
+
+#[test]
+fn test_doi_display() {
+    let doi = Doi {
+        0: "https://doi.org/10.12345/Test-Suffix.01".to_string(),
+    };
+    assert_eq!(format!("{}", doi), "10.12345/Test-Suffix.01");
+}
+
+#[test]
+fn test_orcid_display() {
+    let orcid = Orcid {
+        0: "https://orcid.org/0000-0002-1234-5678".to_string(),
+    };
+    assert_eq!(format!("{}", orcid), "0000-0002-1234-5678");
+}
+
+#[test]
+fn test_timestamp_display() {
+    let stamp: Timestamp = Default::default();
+    assert_eq!(format!("{}", stamp), "1970-01-01 00:00:00");
+}
+
+#[test]
+fn test_doi_fromstr() {
+    let standardised = Doi {
+        0: "https://doi.org/10.12345/Test-Suffix.01".to_string(),
```

Comment:
This (and the likes) could be replaced with `let standardised = Doi("https://doi.org/10.12345/Test-Suffix.01".to_string());`, i.e. `Doi("")` instead of `Doi { 0: "" }`